clients/clients.go

```diff
@@ -316,14 +316,35 @@ func (c *Clients) onRoomMemberEvent(client *mautrix.Client, event *mevt.Event) {
 		})
 		logger.Print("Accepting invite from user")
 
-		content := struct {
+		inviteContent := struct {
 			Inviter id.UserID `json:"inviter"`
 		}{event.Sender}
 
-		if _, err := client.JoinRoom(event.RoomID.String(), "", content); err != nil {
+		if _, err := client.JoinRoom(event.RoomID.String(), "", inviteContent); err != nil {
 			logger.WithError(err).Print("Failed to join room")
-		} else {
-			logger.Print("Joined room")
+			return
+		}
+		logger.Print("Joined room")
+		var rID = id.RoomID(event.RoomID.String())
+		var plContent struct {
+			Users        map[string]int `json:"users"`
+			UsersDefault int
+		}
+		if client.config.MinimumPowerLevel == 0 {
```

Comment:
This should probably be optional?

Reply:
Go defaults unset integer values to 0, which is incredibly annoying in this context.